src/coreclr/debug/daccess/daccess.cpp

```diff
@@ -5491,6 +5492,25 @@ ClrDataAccess::Initialize(void)
     IfFailRet(GetDacGlobalValues());
     IfFailRet(DacGetHostVtPtrs());
 
+    PathString envVar;
+    DWORD len = WszGetEnvironmentVariable(W("DOTNET_ENABLE_CDAC"), envVar);
```

Comment:
Should we use the infrastructure from `clrconfigvalues.h` instead?

Reply:
Let's at least use the `clrconfignocache.h` infra even if we aren't going to use `clrconfigvalues.h` (I don't know if the full version works in the DAC)